topology/Thomas Jefferson National Accelerator Facility/JLAB/Thomas Jefferson National Accelerator Facility GLUEX Glidein.yaml

```diff
@@ -10,8 +10,8 @@ Resources:
           ID: 818acb7ebcd1865a17551799f0cfeb5256e0db20
           Name: Kurt Strosahl
         Secondary:
-          ID: 4065a7a752d68e5ca31f42b6150ca28e946bef5b
-          Name: Sandra Philpott
+          ID: ea9db644eb6f8632cf87fd2cbac8a6e393c21f71 
+          Name: Wesley Moore
```

Comment:
@wmoore28 was this change intentional? I think I missed this on our call

Reply:
I'm going to assume this was intended for now. If it wasn't, @wmoore28, you can submit another PR for that.